.github/workflows/production-2gp.yml

```
@@ -0,0 +1,25 @@
+on:
+  workflow_call:
+    secrets:
+      dev-hub-auth-url:
+        required: True
+
+jobs:
+  upload-production-release-2gp:
+    name: "Upload Production Release (2GP)"
+    environment: packaging-prod
+    concurrency: packaging
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cumulus-actions/release-production-2gp@v1
+        with:
+          packaging-org-auth-url: '${{ secrets.dev-hub-auth-url }}'
+  release-test:
+    name: "Release Test"
+    needs:
+      - upload-production-release-2gp
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: cumulus-actions/release-test@v1
+        with:
+          dev-hub-auth-url: ${{ secrets.dev-hub-auth-url }}
```

```suggestion
          dev-hub-auth-url: '${{ secrets.dev-hub-auth-url }}'
```
lets stay consistent with our interpolation formatting.